ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/EthSignerBaseCommand.java

```diff
@@ -44,6 +48,47 @@
     footer = "EthSigner is licensed under the Apache License 2.0")
 public class EthSignerBaseCommand implements Config {
 
+  static class TlsServerOptions implements TlsOptions {
+
+    @Option(
+        names = "--tls-keystore-file",
+        description =
+            "Path to PKCS#12 format trust store for the JSON-RPC HTTP service client trust.",
```

Comment:
do we need all these words? service client trust at the end sounds weird

Reply:
"Path to PKCS#12 format key store for ?? service."

Reply:
Done